wiki/Ranking_Criteria/osu!/en.md

```diff
@@ -57,27 +54,21 @@ Overall rules and guidelines apply to every kind of osu! difficulty. Rhythm rela
 -   **Hit objects must never be off-screen in 4:3 aspect ratios.** Hit objects that are even partially off-screen can create reading difficulties. Test play your beatmap to confirm this.
 -   **Spinners must be long enough for Auto to achieve 1000 bonus score.** Short spinners are unreasonably difficult to complete.
 -   **Each beatmap must use at least two different custom combo colours unless the default skin is forced.** The combo colours must not blend with the beatmap's background/storyboard/video in any case. This is so hit objects are always visible to the player and custom skin's combo colours do not blend with the background accidentally.
--   **Every slider must have a clear and visible path to follow from start to end.** Sliders which overlap themselves in a way that makes any of their individual sections unreadable or ambiguous cannot be used, such as burai sliders and hold sliders without straightforward slider borders.
+-   **Every slider must have a clear and visible path to follow from start to end.** Sliders that overlap themselves without straightforward slider borders and sliders whose individual sections are unreadable cannot be used. A slider's end position must be clear under the assumption that a player has fully transparent `sliderendcircle.png`/`sliderendcircleoverlay.png` skin elements.
```

Comment:
I disagree with specifically mentioning the skin elements here, as many may still use skins that use the transparent hitcircle and numbers as hitcircles to get the transparent sliderend style, as this is how it was done for years before seperate-end support was finally added. Just a particularity thing.
`A slider's end position must be clear under the assumption that a player has a skin which makes slider end circles invisible.`

Reply:
yes